README.md

```diff
@@ -27,84 +27,91 @@ cargo build --release
 
 ## Run
 
+### Testnets
+
+Centrifuge has multiple testnets online.
+
+1. Fulvous is an ephemeral testnet that is tracking `master`.
```

Comment:
I would change this to: `Fulvous` is an ephemeral testnet for internal dev purposes, and testing internal integrations with all the centrifuge components. Not recommended for external usage due to its nature and purpose.

Reply:
I like that, changed.